### PR TITLE
Fix callout formatting

### PR DESCRIPTION
**Pull request by misteryeo**
I think the space caused the formatting to not work.
Amazon Seller Partner  Airbyte Documentation 2022-08-01 at 8 41 44 AM

### CODE REVIEW

**Review by YowanR — CHANGES_REQUESTED** [review]
Thanks for calling this out! You have a typo -- extra a in tracked :)

File: docs/integrations/sources/amazon-seller-partner.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
 This page guides you through the process of setting up the Amazon Seller Partner source connector.
 
-::: caution
+:::caution
 
-Known blocking issue with this connector tracked here: https://github.com/airbytehq/airbyte/issues/14734
+Authenticating this Alpha connector is currently blocked. This is a known issue being traacked here: https://github.com/airbytehq/airbyte/issues/14734
```
Comment (Amruta-Ranade) [inline docs/integrations/sources/amazon-seller-partner.md:7]: nit: "traacked" > "tracked"

**Review by Amruta-Ranade — APPROVED** [review]
Thank you!

**Review by YowanR — APPROVED** [review]
LGTM -- thanks!

**Pull request merged**